src/main/java/org/threeten/extra/chrono/InternationalFixedChronology.java

```diff
@@ -0,0 +1,490 @@
+     * Earlier eras must have sequentially lower values.
+     * Each chronology must refer to an enum or similar singleton to provide the era values.
+     * <p>
+     * This method returns the singleton era of the correct type for the specified era value.
+     *
+     * @param eraValue the era value
+     * @return the calendar system era, not null
+     * @throws DateTimeException if unable to create the era
+     */
+    @Override
+    public InternationalFixedEra eraOf(final int eraValue) {
+        return InternationalFixedEra.of(eraValue);
+    }
+
+    /**
+     * Gets the list of eras for the chronology.
+     * <p>
+     * Most calendar systems have an era, within which the year has meaning.
+     * If the calendar system does not support the concept of eras, an empty
+     * list must be returned.
+     *
+     * @return the list of eras for the chronology, may be immutable, not null
+     */
+    @Override
+    public List<Era> eras() {
+        return Arrays.<Era>asList(InternationalFixedEra.values());
+    }
+
+    //-----------------------------------------------------------------------
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ValueRange range(final ChronoField field) {
+        switch (field) {
+            case ALIGNED_DAY_OF_WEEK_IN_YEAR:
+                return ALIGNED_DAY_OF_WEEK_RANGE;
+            case ALIGNED_DAY_OF_WEEK_IN_MONTH:
+            case DAY_OF_WEEK:
+                return DAY_OF_WEEK_RANGE;
+            case ALIGNED_WEEK_OF_MONTH:
+                return WEEK_OF_MONTH_RANGE;
+            case ALIGNED_WEEK_OF_YEAR:
+                return WEEK_OF_YEAR_RANGE;
+            case DAY_OF_MONTH:
+                return DAY_OF_MONTH_RANGE;
+            case DAY_OF_YEAR:
+                return DAY_OF_YEAR_RANGE;
+            case EPOCH_DAY:
+                return EPOCH_DAY_RANGE;
+            case ERA:
+                return ERA_RANGE;
+            case MONTH_OF_YEAR:
+                return MONTH_OF_YEAR_RANGE;
+            case PROLEPTIC_MONTH:
+                return PROLEPTIC_MONTH_RANGE;
+            case YEAR_OF_ERA:
+            case YEAR:
+                return YEAR_RANGE;
+            default:
+                return field.range();
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public int prolepticYear(final Era era, final int yearOfEra) {
+        if (!(era instanceof InternationalFixedEra)) {
+            throw new ClassCastException("Invalid era: " + era);
+        }
+
+        if (yearOfEra < 1) {
+            throw new DateTimeException("Invalid year of era: " + yearOfEra);
+        }
+
+        return yearOfEra;
+    }
+
+    /**
+     * Get the count of leap years since International fixed year 1.
+     * <p/>
+     *
+     * @param prolepticYear The year.
+     * @return The number of leap years since International fixed year 1.
+     */
+    public static long getLeapYearsBefore(final long prolepticYear) {
```

Comment:
No, I meant just make it package private.  External consumers shouldn't touch it.